fearless/Modules/Wallet/History/View/HistoryItemTableViewCell.swift

```diff
@@ -1,33 +1,162 @@
-    @IBOutlet private var statusImageView: UIImageView!
-    @IBOutlet private var trailingConstraint: NSLayoutConstraint!
+    private let transactionTypeView = UIImageView()
+
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = .p1Paragraph
+        label.textColor = R.color.colorWhite()
+        label.lineBreakMode = .byTruncatingMiddle
+        return label
+    }()
+
+    private let subtitleLabel: UILabel = {
+        let label = UILabel()
+        label.font = .p2Paragraph
+        label.textColor = R.color.colorTransparentText()
+        return label
+    }()
+
+    private let amountLabel: UILabel = {
+        let label = UILabel()
+        label.font = .p1Paragraph
+        return label
+    }()
+
+    private let timeLabel: UILabel = {
+        let label = UILabel()
+        label.font = .p2Paragraph
+        label.textColor = R.color.colorTransparentText()
+        return label
+    }()
+
+    private var statusImageView: UIImageView?
 
     var viewModel: WalletViewModelProtocol?
 
-    override func prepareForReuse() {
-        super.prepareForReuse()
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        configure()
+    }
 
-        (viewModel as? HistoryItemViewModel)?.imageViewModel?.cancel()
+    @available(*, unavailable)
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
     }
 
-    override func awakeFromNib() {
-        super.awakeFromNib()
+    private func configure() {
+        backgroundColor = .clear
+
+        selectedBackgroundView = UIView()
+        selectedBackgroundView?.backgroundColor = R.color.colorCellSelection()!
 
-        let selectedBackgroundView = UIView()
-        selectedBackgroundView.backgroundColor = R.color.colorAccent()!.withAlphaComponent(0.3)
-        self.selectedBackgroundView = selectedBackgroundView
+        setupLayout()
+    }
+
+    private func setupLayout() {
+        contentView.addSubview(transactionTypeView)
+
+        transactionTypeView.snp.makeConstraints { make in
+            make.leading.equalTo(contentView).offset(UIConstants.horizontalInset)
+            make.centerY.equalTo(contentView)
+            make.width.equalTo(Constants.iconSize)
+        }
+
+        contentView.addSubview(titleLabel)
+
+        titleLabel.snp.makeConstraints { make in
+            make.leading.equalTo(transactionTypeView.snp.trailing).offset(UIConstants.horizontalInset / 2.0)
+            make.top.equalTo(contentView).offset(Constants.verticalInset)
+        }
+
+        contentView.addSubview(subtitleLabel)
+
+        subtitleLabel.snp.makeConstraints { make in
+            make.leading.equalTo(transactionTypeView.snp.trailing).offset(UIConstants.horizontalInset / 2.0)
+            make.top.equalTo(titleLabel.snp.bottom)
+            make.bottom.equalTo(contentView).offset(-Constants.verticalInset)
```

Comment:
inset(Constants.verticalInset)
`offset(..)` is used with views in the same view hierarchy, i.e.  `make.leading.equalTo(transactionTypeView.snp.trailing).offset(..)`

Reply:
fixed